models/tasks.js

```diff
@@ -14,6 +14,9 @@ const { TASK_TYPE, TASK_STATUS } = require("../constants/tasks");
 const updateTask = async (taskData, taskId = null) => {
   try {
     taskData = await toFirestoreData(taskData);
+    if (taskData.userNotFound) {
```

Comment:
Should this be here? Should the model be silently returning, instead of updating the Firestore?

Reply:
Thanks for pointing it out. Moved this check to `controller/tasks.js`